app/dashboard/organization/overview/page.tsx

```diff
@@ -41,7 +44,8 @@ export default function OrganizationOverview() {
     const [userOrgID, setUserOrgID] = useState(-1);
     const [loading, setLoading] = useState(true);
     const [isOwner, setIsOwner] = useState(null);
```

Comment:
For some reason I can't highlight multiple lines and group them as a single comment, so same comment applies here